android/app/src/main/java/com/votlin/android/ui/fragment/TalksListFragment.kt

```diff
@@ -41,56 +41,52 @@ class TalksListFragment : RootFragment<TalksListView>(), TalksListView {
     override val fragmentModule: Kodein.Module = Kodein.Module {
         bind<TalksListPresenter>() with provider {
             TalksListPresenter(
-                    view = this@TalksListFragment,
-                    errorHandler = AndroidErrorHandler(),
-                    executor = instance(),
-                    repository = instance())
+                view = this@TalksListFragment,
+                errorHandler = AndroidErrorHandler(),
+                executor = instance(),
+                repository = instance()
+            )
         }
     }
 
-    private val adapter = TalksAdapter { presenter.onTalkClicked(it) }
+    private lateinit var talksAdapter: TalksAdapter
 
     override fun initializeUI() {
-        talks.layoutManager = LinearLayoutManager(activity, LinearLayoutManager.VERTICAL, false)
-        talks.adapter = adapter
+        talksAdapter = TalksAdapter { presenter.onTalkClicked(it) }
+        talks.apply {
+            layoutManager = LinearLayoutManager(activity, LinearLayoutManager.VERTICAL, false)
+            adapter = talksAdapter
+        }
     }
 
     override fun registerListeners() {
         // Nothing to do here
     }
 
-    override fun showProgress() = progressView.showMe()
+    override fun showProgress() {
+        progressView.showMe()
```

Comment:
We don't need here the body of the function!

Reply:
I have a rule for expression body: I only use it for functions that return something. It's more clear for me in this way. However, this is a personal taste, so, I'm going to revert this change if you don't like it.